app/src/main/res/layout-land/app_language_activity.xml

```diff
@@ -15,19 +15,10 @@
       app:layout_constraintStart_toStartOf="parent"
       app:layout_constraintTop_toTopOf="parent">
 
-      <androidx.appcompat.widget.Toolbar
-        android:id="@+id/app_language_toolbar"
-        android:layout_width="match_parent"
-        android:layout_height="?attr/actionBarSize"
-        android:background="?attr/colorPrimary"
-        android:fontFamily="sans-serif"
-        android:minHeight="?attr/actionBarSize"
-        android:textSize="20sp"
-        app:navigationContentDescription="@string/go_to_previous_page"
-        app:navigationIcon="?attr/homeAsUpIndicator"
-        app:title="@string/title_app_language"
-        app:titleTextAppearance="@style/ToolbarTextAppearance"
-        app:titleTextColor="@color/white" />
+      <include
```

Comment:
Create a landscape toolbar with height 56dp and use it in all landscape layouts. As it is looking smaller in size.

Reply:
Done